<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TGo4CreateNewCondition</class>
  <widget class="QDialog" name="TGo4CreateNewCondition" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>234</width>
        <height>161</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Create condition</string>
    </property>
    <property name="windowIcon" >
      <iconset>:/icons/go4logo2_big.png</iconset>
    </property>
    <property name="toolTip" stdset="0" >
      <string/>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="QLabel" name="NameLabel" >
          <property name="text" >
            <string>Name:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="CondName" >
          <property name="text" >
            <string>enter condition name</string>
          </property>
          <property name="modified" >
            <bool>false</bool>
          </property>
          <property name="dragEnabled" >
            <bool>true</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Name of new condition</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="TypeLabel" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text" >
            <string>Type:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="1" colspan="2" >
        <widget class="QComboBox" name="ClassnameCombo" >
          <property name="insertPolicy" >
            <enum>QComboBox::InsertAtCurrent</enum>
          </property>
          <property name="duplicatesEnabled" >
            <bool>false</bool>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Type of condition</string>
          </property>
          <item>
            <property name="text" >
              <string>1-D window</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>2-D window</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Polygon (TCutG) </string>
            </property>
          </item>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="QLabel" name="ArraySizeLbl" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text" >
            <string>Array size:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="2" >
        <widget class="QSpinBox" name="ArraySizeSpin" >
          <property name="enabled" >
            <bool>true</bool>
          </property>
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="specialValueText" >
            <string>No array</string>
          </property>
          <property name="buttonSymbols" >
            <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum" >
            <number>100</number>
          </property>
          <property name="minimum" >
            <number>1</number>
          </property>
          <property name="singleStep" >
            <number>5</number>
          </property>
          <property name="value" >
            <number>1</number>
          </property>
          <property name="toolTip" stdset="0" >
            <string>Define size of condition array</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="3" column="0" colspan="3" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QPushButton" name="RemoteBtn" >
              <property name="minimumSize" >
                <size>
                  <width>70</width>
                  <height>30</height>
                </size>
              </property>
              <property name="text" >
                <string>&amp;Create remote</string>
              </property>
              <property name="shortcut" >
                <string>Alt+C</string>
              </property>
              <property name="autoDefault" >
                <bool>false</bool>
              </property>
              <property name="default" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer187" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="CloseWin" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>30</width>
                  <height>30</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="icon" >
                <iconset>:/icons/close16.png</iconset>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Close editor without change.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections>
    <connection>
      <sender>CloseWin</sender>
      <signal>clicked()</signal>
      <receiver>TGo4CreateNewCondition</receiver>
      <slot>close()</slot>
    </connection>
    <connection>
      <sender>RemoteBtn</sender>
      <signal>clicked()</signal>
      <receiver>TGo4CreateNewCondition</receiver>
      <slot>accept()</slot>
    </connection>
  </connections>
</ui>
